app/javascript/state/fundraiser/actions.js

```diff
@@ -13,6 +13,7 @@ export function changeCurrency(payload: string): FundraiserAction {
 }
 
 export function setSubmitting(payload: boolean): FundraiserAction {
+  $.publish('fundraiser:set_submitting', [payload]);
```

Comment:
This would be cleaner with middleware? 

```js
const amplitudeMiddleware = state => next => action => {
  const { type, ...rest } = action;
  logEvent(type, rest);
  return next(action);
}

Reply:
You'd also not need `$.publish/$.subscribe` anymore for redux actions, which would simplify `subscribeEvents`